runtime/makelib/targets.mk.zos.inc.ftl

```diff
@@ -112,6 +112,14 @@ ifeq ($(UMA_TARGET_NAME),jvmti_test_src)
   UMA_ZOS_FLAGS += -Wc,DLL,EXPORTALL
 endif
```

Comment:
Unrelated to this PR I noticed this block of code. @keithc-ca is this something that should belong in a `makefilestub` for the `jvmti_test_agent` and `jvmti_test_src` tests? Seems odd to put it here.

Reply:
I never noticed that before, but I'm inclined to agree: those flags belong nearer to `jvmti_test_src`.

Reply:
Fixed in #6622.